### PR TITLE
updated concepts real-time executor

### PR DESCRIPTION
**Pull request by JanStaschulat**

### CODE REVIEW

File: _docs/concepts/client_library/real-time_executor/index.md [hunk 1]
```diff
@@ -506,26 +507,19 @@ In this section, we describe the necessary changes to the Executor API:
...
-
-### Meta Executor Concept
-
-The idea of the Meta Executor is to abstract away the callback-group assignment, thread allocation and other inner workings of the Executors from the user, thereby presenting a simple API that resembles the original Executor interface. Internally, the Meta Executor maintains multiple instances of our Callback-group-level Executor (Cbg-Executor).
-
-The Meta Executor internally binds these Executors to the underlying kernel threads, assigns them a priority, chooses the scheduling mechanism (e.g., SCHED-FIFO policy) and then dispatches them. When adding a node with its list of callback group and real-time profiles to the Meta Executor, it parses the real-time profiles and assigns the node’s callback groups to the relevant internal Executors.
-
-![Illustration of the Meta-Executor concept](meta-executor_concept.png)
+The callback-grouplevel executor has been merged into ROS 2 rclcpp in [pull request 1218](https://github.com/ros2/rclcpp/pull/1218/commits).
```
Comment (ralph-lange) [inline _docs/concepts/client_library/real-time_executor/index.md:510]:
```suggestion
The callback-group-level executor has been merged into ROS 2 rclcpp in [pull request 1218](https://github.com/ros2/rclcpp/pull/1218/commits).
```

File: _docs/concepts/client_library/real-time_executor/index.md [hunk 2]
```diff
@@ -610,32 +604,16 @@ doi: 10.1109/SIMPAR.2018.8376277
...
-* In-depth runtime analysis of default rclcpp Executor using ROS 2 tracetools developed in micro-ROS.
-* Design and implementation of static Executor for rclcpp with improved performance.
-* Research of concepts for model-based optimization of end-to-end latencies.
-
-2020
+## References
 
-* Integration with a selected advanced scheduling and resource monitoring mechanisms such as reservation-based scheduling.
-* Integration with selected sleep modes and low-power modes.
+* [L2020]<a name="L20202"></a> Ralph Lange: Advanced Execution Management with ROS 2, ROS-Industrial Conference, Dec 2020 [[Slides]](https://micro-ros.github.io/download/2020-12-16_Advanced_Execution_Management_with_ROS_2.pdf)
```
Comment (ralph-lange) [inline _docs/concepts/client_library/real-time_executor/index.md:610]: Is the 20202 intended?

**Review by ralph-lange — APPROVED** [review]
Two minor findings. Approved it already.

File: _docs/concepts/client_library/real-time_executor/index.md (hunk 2 above)
Comment (JanStaschulat) [reply _docs/concepts/client_library/real-time_executor/index.md]: Fixed typo.

**Review by ralph-lange — APPROVED** [review]

**Comment by JanStaschulat** [comment]
@ralph-lange could you merge the PR? I am not authorized.

**Comment by ralph-lange** [comment]
Seems to be a temporary issues with Twitter only. If I click the links, they work. Will merge this PR therefore.

**Pull request merged**